app/views/layouts/_header.html.erb

```diff
@@ -1,58 +1,87 @@
-<nav class="navbar navbar-expand-lg navbar-light bg-white">
+<nav class="navbar navbar-expand-lg bg-white navbar-light">
   <a class="navbar-brand" href="/">
-    <%= image_tag("cvlogo.svg", class: "logo", alt: "CircuitVerse Logo") %>
+    <%= image_tag("CircuitVerse.png", class: "navbar-logo", alt: "CircuitVerse Logo") %>
   </a>
-  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#collapsedNavbar"
-          aria-controls="collapsedNavbar" aria-expanded="false" aria-label="Toggle navigation">
+  <button class="navbar-toggler navbar-light" type="button" data-toggle="collapse" data-target="#collapsed-navbar"
+  aria-controls="collapsedNavbar" aria-expanded="false" aria-label="Toggle navigation">
     <span class="navbar-toggler-icon"></span>
   </button>
-
-  <div class="collapse navbar-collapse" id="collapsedNavbar">
-    <ul class="navbar-nav ml-auto mt-2 mt-lg-0 navigationBar-item">
-      <li class="nav-item px-1">
-        <a class="btn btn-outline-success rounded-0 simulator text-dark " href="/simulator">Simulator</a>
+  <div class="collapse navbar-collapse" id="collapsed-navbar">
+    <ul class="navbar-nav ml-auto navigationBar-item">
+      <li class="nav-item px-1 pt-1">
+        <div class="row-text navbar-search-icon-container">
```

Comment:
This is not navigable by keyboard. Add tabindex="0" and aria-label="search"

Reply:
Done

Reply:
Mistakenly added; removed now